tilings/tiling.py

```diff
@@ -945,7 +951,9 @@ def component_fusion(self, row=None, col=None):
         """
         return self._fusion(row, col, ComponentFusion)
 
-    def sub_tiling(self, cells: Iterable[Cell], factors: bool = False) -> "Tiling":
+    def sub_tiling(
+        self, cells: Iterable[Cell], factors: bool = False, add_assumptions=tuple()
```

Comment:
Can you add the typing for the `add_assumptions` argument?

Reply:
Should be a Tuple[TrackingAssumptions]